### PR TITLE
Lightens the punishment for saying 'rouge' and fixes silicon brain damage

### PR DESCRIPTION
**Pull request by PopNotes**
:cl: PopNotes
add: Using emotes IC gives you some brain damage and a stun.
fix: Lightened the punishment for saying 'rouge' (see above) and fixed silicons getting brain damage from saying it.
/:cl:

### CODE REVIEW

**Comment by tortellinitony** [comment]
why is this a thing in the first place what the fuck

**Comment by tortellinitony** [comment]
"Rouge" is a color, you should feel like a moron because you're punishing people for saying a color in chat?

File: hippiestation/code/modules/mob/living/say.dm [hunk 1]
```diff
@@ -1,6 +1,13 @@
 /mob/living/say(message, bubble_type,var/list/spans = list(), sanitize = TRUE, datum/language/language = null)
 	if(findtext(message, "rouge"))
-		to_chat(src, "<span class='boldwarning'>You feel like a fucking moron.</span>")
-		adjustBrainLoss(60)
-		src.playsound_local(get_turf(src), 'hippiestation/sound/misc/slidewhistle_down.ogg', 100, FALSE, pressure_affected = FALSE)
-	..()
+		if(!issilicon(src))
+			to_chat(src, "<span class='boldwarning'>You feel like a fucking moron.</span>")
+			adjustBrainLoss(60)
+			src.playsound_local(get_turf(src), 'hippiestation/sound/misc/slidewhistle_down.ogg', 100, FALSE, pressure_affected = FALSE)
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/mob/living/say.dm]: you only need this line once unindented after the else clause

File: hippiestation/code/modules/mob/living/say.dm [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
+		if(!issilicon(src))
+			to_chat(src, "<span class='boldwarning'>You feel like a fucking moron.</span>")
+			adjustBrainLoss(60)
+			src.playsound_local(get_turf(src), 'hippiestation/sound/misc/slidewhistle_down.ogg', 100, FALSE, pressure_affected = FALSE)
+		else
+			to_chat(src, "<span class='boldwarning'>Your own stupidity overloads your sensors.</span>")
+			src.Knockdown(rand(80,120))
+			src.confused += 5
+			src.flash_act(affect_silicon = 1)
+			src.playsound_local(get_turf(src), 'hippiestation/sound/misc/slidewhistle_down.ogg', 100, FALSE, pressure_affected = FALSE)
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/mob/living/say.dm]: You only need this line once unindented after the else clause

File: hippiestation/code/modules/mob/living/say.dm (hunk 1 above)
Comment (GuyonBroadway) [inline hippiestation/code/modules/mob/living/say.dm]: You only need to play this sound once after the code under the else clauses has been executed.

File: hippiestation/code/modules/mob/living/say.dm (hunk 2 above)
Comment (GuyonBroadway) [inline hippiestation/code/modules/mob/living/say.dm]: You only need to play this sound once after the code under the else clauses has been executed.

**Comment by GuyonBroadway** [comment]
Code is now good, this still probably needs admin input.

**Pull request closed without merging**